ocf/client.md

```diff
@@ -87,6 +88,8 @@ Extends `ResourceId`. Used for creating and registering resources, exposes the p
 
  The `interfaces` property MUST at least contain `"oic.if.baseline"`.
 
+ The `endpoints` property is an array of strings. Each string is a URI consisting of an origin (scheme and authority). It MUST contain at least one endpoint.
```

Comment:
I think this should array of "ep" objects.  Something like: ` "eps": [{"ep": "coap://[fe80::b1d6]:1111", "pri": 2}, {"ep": "coaps://[fe80::b1d6]:1122"}]`

Reply:
Here is the spec about Endpoint information:

Endpoint is represented by Endpoint information which consists of two items of key-value pair, "ep" and "pri".

- "ep" shall have as its value a URI with the scheme component indicating Transport Protocol Suite and the authority component indicating the Endpoint Locator

- “pri”- When there are multiple Endpoints, "pri" indicates the priority among them. "pri" shall be represented as a positive integer (e.g. "pri": 1) and the lower the value, the higher the priority. The default "pri" value is 1, i.e. when "pri" is not present, it shall be equivalent to "pri": 1.

Reply:
This makes sense to me.